docs/topics/api/overview.rst

```diff
@@ -314,6 +314,7 @@ these docs.
 v4 API changelog
 ----------------
 
+* 2019-11-14: removed ``is_source_public`` property from addons API https://github.com/mozilla/addons-server/issues/12514
```

Comment:
If you look at dates you can see they're in descending order, so the new entry should be at the end of the list rather than the top

Reply:
Oh, Sorry.
I submit patch again.